<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CBuksanSpy</class>
 <widget class="QMainWindow" name="CBuksanSpy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BuksanSpy</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton_2">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::ContextMenuPolicy::ActionsContextMenu</enum>
      </property>
      <property name="text">
       <string>Start recording</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton_4">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Start recording 
 via selected device</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="6">
     <layout class="QGridLayout" name="CamLayout">
      <item row="0" column="0">
       <widget class="QTabWidget" name="Cams_TabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="AllCams">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <attribute name="title">
          <string>All</string>
         </attribute>
         <widget class="QWidget" name="gridLayoutWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>651</width>
            <height>401</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="camslay"/>
         </widget>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Page</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="pushButton_3">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Media journal</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>898</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="InitCams_action"/>
    <addaction name="StartAllCams_Action"/>
    <addaction name="StartCurrentCam_Action"/>
    <addaction name="Quit_Action"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Media</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Administration</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>Users and groups</string>
     </property>
     <addaction name="users_Action"/>
    </widget>
    <widget class="QMenu" name="menu_5">
     <property name="title">
      <string>Devices</string>
     </property>
     <addaction name="ShowDevList_Action"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="menu_4"/>
    <addaction name="separator"/>
    <addaction name="menu_5"/>
    <addaction name="eventlog_Action"/>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_help"/>
    <addaction name="action_Qt"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_6"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="StartAllCams_Action">
   <property name="text">
    <string>Start recording</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="StartCurrentCam_Action">
   <property name="text">
    <string>Start recording via selected device</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="Quit_Action">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Q</string>
   </property>
  </action>
  <action name="users_Action">
   <property name="text">
    <string>Users</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+U</string>
   </property>
  </action>
  <action name="MandatoryGroups_Action">
   <property name="text">
    <string>Mandate groups</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+M</string>
   </property>
  </action>
  <action name="AddDevice_Action">
   <property name="text">
    <string>Add device</string>
   </property>
  </action>
  <action name="InitCams_action">
   <property name="text">
    <string>Init devices</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+I</string>
   </property>
  </action>
  <action name="AddDev_Action">
   <property name="text">
    <string>Add device</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="ShowDevList_Action">
   <property name="text">
    <string>Show devices list</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="eventlog_Action">
   <property name="text">
    <string>Event journal</string>
   </property>
   <property name="toolTip">
    <string>Event journal</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+L</string>
   </property>
  </action>
  <action name="action_help">
   <property name="text">
    <string>User manual</string>
   </property>
  </action>
  <action name="action_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="toolTip">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
